templates/partials/ie-chart.html

```diff
@@ -2,26 +2,24 @@
 {% set reports = committee.reports  %}
 {% set totals = committee.totals  %}
 {% import 'macros/charts.html' as charts %}
-{% import 'macros/null.html' as null %}
-{% macro group_bar_tooltip(value, key) %}
-<div class="chart-series__bar__tooltip">
-  <h5 class="chart-series__bar__tooltip__title">{{ key|replace('_', ' ')|title }}</h5>
-  <span class="chart-series__bar__tooltip__value">{{ null.null( value | currency )}}</span>
-</div>
-{% endmacro %}
 
 {% if series_group_has_data(reports, ('independent_contributions_period', 'independent_expenditures_period')) %}
```

Comment:
Just noting that the variable from the API (`independent_contributions_period`) must also be wrong, so we'll want to update that eventually.